raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -194,6 +200,18 @@ contract TokenNetwork is Utils {
         public
         returns (bytes32)
     {
+        channel_counter += 1;
```

Comment:
The global counter is used instead of a per channel counter to save gas by deleting entries from `participants_hash_to_channel_counter`, right?

Reply:
Correct, but we would also have redundant data in storage, which is not a good practice (all per channel counters for settled channels, that will not be reopened).